packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_sidebar.dart

```diff
@@ -0,0 +1,277 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class PropertyEditorSidebar extends StatelessWidget {
+  const PropertyEditorSidebar({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text('Property Editor', style: theme.textTheme.titleMedium),
```

Comment:
since we only reference theme once, we can remove the local variable by looking up Theme.of(context) directly here

Reply:
Done!